### DISCUSSION TITLE
Default seed in config.txt

### DISCUSSION THREAD

**Discussion by realm174** [lllyasviel/Fooocus#1673] [Q&A] [answered]
Hello, I created a particular image I like, as a starting point, so I figured I could use config.txt to set the same settings i used to generate that image. And that works well, with one exception, and that's the seed. I couldn't see anything in the config_modification_tutorial.txt file about the seed either, unless I missed it. Any suggestion?

Thanks!

**Comment by mashb1t** [comment, marked as answer]
Sorry to say, this is currently not possible, but already mentioned in https://github.com/lllyasviel/Fooocus/issues/1692#issuecomment-1873102351 as an enhancement for the future as well as preset creation from the UI.
EDIT: Suggestion - use the seed as default value in the code and set seed random checkbox to false by default
